src/styles/card.scss

```diff
@@ -0,0 +1,78 @@
+.card {
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: $white-color;
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 0 16px 16px;
+
+  &__image {
+    display: flex;
+    justify-content: center;
+
+    &-detail {
+      width: 160px;
+      height: 134px;
+      margin-block: 32px 40px;
+    }
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__prodcode {
+    color: $grey-color;
+    font-size: 10px;
+    line-height: 14px;
+    padding-top: 4px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    padding-bottom: 16px;
+    padding-block: 24px 16px;
+  }
+
+  &__price-name {
+    color: $grey-color;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+  }
```

Comment:
```suggestion
    &-name {
      color: $grey-color;
      font-size: 12px;
      line-height: 18px;
    }
  
    &-value {
      font-weight: 700;
      font-size: 16px;
      line-height: 18px;
    }
  }
```

Reply:
> 

It looks optional for me, but at least reply to comments that a reliever leave